deploy/base/Dockerfile.builder

```
@@ -0,0 +1,16 @@
+FROM rust:1.88-slim-bookworm 
+
+LABEL maintainer="UnifyAir <support@unifyair.com>"
+
+# Install build dependencies and cargo-chef
+RUN apt-get update && apt-get install -y \
```

RuboCop (1.76.1)

[fatal] unexpected token tIDENTIFIER
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)